packages/rainbowkit/src/components/ConnectButton/ConnectButtonRenderer.tsx

```diff
@@ -80,9 +81,13 @@ export function ConnectButtonRenderer({
 
   const { data: ensAvatar } = useEnsAvatar({
     addressOrName: accountData?.address,
+    chainId: chainMetadataByName.mainnet?.chainId,
```

Comment:
Nit: you could just hardcode `1` here and not have to export/import other component... 🤔